src/style/foundation/video-calling.less

```diff
@@ -167,7 +167,7 @@
       border-radius: 1rem;
       font-size: 1.5rem;
       &:hover {
-        background-color: var(--inactive-call-button-border);
+        background-color: var(--inactive-call-button-hover-bg);
```

Comment:
more inline with the rest of the call ui button it does change the light mode hover state (couldn't find hover states on figma)
Before:
![image](https://github.com/user-attachments/assets/221c0d39-aee4-46ab-a38c-f44606601f13)
After:
![image](https://github.com/user-attachments/assets/36c2a58f-3db1-4ebb-b62e-ac3d981b3826)